src/plots/utils.py

```diff
@@ -0,0 +1,10 @@
+line_color = [
```

Comment:
this is a good idea, we could probably add other utils in here for other pages, like the colour_per_model in `forecast.py`

Reply:
yea good idea

Reply:
ill move that